dev-tools/packaging/templates/ironbank/auditbeat/hardening_manifest.yaml

```diff
@@ -14,7 +14,7 @@ tags:
 # Build args passed to Dockerfile ARGs
 args:
   BASE_IMAGE: "redhat/ubi/ubi9"
-  BASE_TAG: "9.6"
+  BASE_TAG: "\"9.7\""
```

Comment:
@v1v looks like something needs to be fixed in the automation right?

Reply:
It's also interesting taht our e2e tests work as expected:
- https://github.com/elastic/oblt-updatecli-policies/pull/141/files#diff-370873feed803e1161c1debf10956878dbcadf682d8495ba0147423b7fe100e1